code_samples/front/render_page/config/packages/ezplatform_page_fieldtype.yaml

```diff
@@ -0,0 +1,19 @@
+ezplatform_page_fieldtype:
+    layouts:
+        sidebar:
+            identifier: sidebar
+            name: Right sidebar
+            description: Main section with sidebar on the right
+            thumbnail: /assets/images/layouts/sidebar.png
+            template: '@ezdesign/layouts/sidebar.html.twig'
+            zones:
+                first:
+                    name: First zone
+                second:
+                    name: Second zone
+    blocks:
+        contentlist:
+            views:
+                custom:
+                    template: '@ezdesign/blocks/contentlist.html.twig'
+                    name: Custom content list
```

Comment:
Missing newline will make `git` unhappy when adding new lines at the end :smile: